crates/wasmtime/Cargo.toml

```diff
@@ -20,7 +20,7 @@ anyhow = "1.0.19"
 region = "2.0.0"
 libc = "0.2"
 cfg-if = "0.1.9"
-backtrace = "0.3.42"
+backtrace = "=0.3.46"
```

Comment:
Instead of checking in an `=` dependency could we leave the lock file as what's a known good version to test against? That would basically just involve dropping this from the diff. I can look into what's going on with the most recent version of backtrace soon too hopefully.

Reply:
yep, I used version "0.3.46" from Cargo.lock

Reply:
Sure, just amended the commit to remove that.